ci-operator/config/openshift-assisted/assisted-chat-rag/OWNERS

```diff
@@ -0,0 +1,19 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/openshift-assisted/assisted-service-mcp root OWNERS
```

Comment:
This is probably wrong ...

I would expect this to be fetched from the rag repo.